abrt.spec.in

```diff
@@ -43,7 +43,7 @@
     %define docdirversion -%{version}
 %endif
 
-%define libreport_ver 2.10.0
```

Comment:
The PR will fail until the PR in libreport is merged and new libreport is released.